index.html

```diff
@@ -0,0 +1,600 @@
+        <p class="text-center">
+          Hackathons are wonderful experiences to build something amazing
+          literally overnight. Students get to network, learn, collect cool
+          sponsor swag, make friends, and get free food all while being 100%
+          sleep deprived. This is an experience that anyone interested in
+          creative problem solving should not miss.
+          <br /><br />
+          We are committed to building a prosperous community and having fun
+          while doing it.
+        </p>
+      </div>
+    </div>
+  </section>
+  <hr>
+  <!--  Testimonials -->
+  <section id="testimonials" class="container mt-5 mb-5 pt-2 pb-5">
+    <h2 class="text-center mb-3"># Create Something Amazing</h2>
+    <div class="container  align-items-center">
+      <div class="row align-items-center">
+
+        <div class="card test1 mr-3" style="width: 22rem;">
+          <img class="card-img-top" src="static\Kyle_pic.PNG" alt="Card image cap">
+          <div class="card-body">
+            <p class="card-text">It was great to see so many people passionate about designing interesting projects!
+              Everyone was willing to help each other, including people from other teams, providing a great sense of
+              community.</p>
+            <br>
+            <p class="card-text">~ Kyle Yarwood</p>
+          </div>
+        </div>
+
+        <div class="card test2" style="width: 22rem;">
+          <img class="card-img-top img1" src="static\Lavisha_pic.PNG" alt="Card image cap">
+          <div class="card-body">
+            <p class="card-text">Competing in The GoldenHack gave me the opportunity to practice my technical and
+              business skills that I learned in the classroom. I was also able to meet like-minded individuals who
+              shared an interest for business and technology like me!</p>
+            <p class="card-text">~ Lavisha Bugreja</p>
+          </div>
+        </div>
+
+        <div class="card test3 ml-3" style="width: 22rem;">
+          <img class="card-img-top img2" src="static\Ishani_pic.PNG" alt="Card image cap">
+          <div class="card-body">
+            <p class="card-text">The golden hack was the first hackathon I ever attended. The 24 hours helped me push
+              myself both mentally and physically! The uniqueness of the challenges and the variety of workshops made
+              the experience even more memorable!</p>
+            <br>
+            <p class="card-text">~ Ishani Sootha</p>
+          </div>
+        </div>
+
+
+      </div>
+    </div>
+
+  </section>
+  <hr>
+
+  <section id="statistics">
+    <h2 class="text-center mb-3"># Last Year we Had...</h2>
+    <div class="container align-items-center">
+      <div class = "row align-items-center">
+        <div class = "col align-items-center" style="text-align: center;">
+          <div class="circle" style="color: white; background: #053E5E;">

```

Comment:
technically this styling should be in a css class of its own but eh it's fine we'll have a new version of the website up soon enough

Reply:
Ohh I gotcha now. I'll keep that in mind as we go on.